lib/standard/fix_ints.nit

```diff
@@ -0,0 +1,589 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# This file is free software, which comes along with NIT.  This software is
+# distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without  even  the implied warranty of  MERCHANTABILITY or  FITNESS FOR A
+# PARTICULAR PURPOSE.  You can modify it is you want,  provided this header
+# is kept unaltered, and a notification of the changes is added.
+# You  are  allowed  to  redistribute it and sell it, alone or is a part of
+# another product.
+
+# Integers of fixed-size with C-like semantics
+module fix_ints
```

Comment:
You should explain for the module:

* what problem it solves
* how to use it (including specification and rules)
* what is the implementation strategy